examples/http-send-header/http_send_header.bal

```diff
@@ -0,0 +1,18 @@
+import ballerina/http;
+
+type Album readonly & record {|
+    string title;
+    string artist;
+|};
+
+table<Album> key(title) albums = table [
+    {title: "Blue Train", artist: "John Coltrane"},
+    {title: "Jeru", artist: "Gerry Mulligan"}
+];
+
+service / on new http:Listener(9090) {
+
+    resource function get albums() returns record {|*http:Ok; Album[] body;|} {
+        return {body: albums.toArray(), headers:{"x-music-genre":"Jazz"}};
+    }
+}
```

Comment:
Lets use the below code as the above is not OpenAPI friendly. 

```ballerina
import ballerina/http;

type Album readonly & record {|
    string title;
    string artist;
|};

table<Album> key(title) albums = table [
    {title: "Blue Train", artist: "John Coltrane"},
    {title: "Jeru", artist: "Gerry Mulligan"}
];

type AlbumOk record {|
    *http:Ok;
    record {|
        string x\-music\-genre;
    |} headers;
    Album[] body;
|};

service / on new http:Listener(9090) {

    resource function get albums() returns AlbumOk {
        return {
            headers: {
                x\-music\-genre: "jazz"
            }, 
            body: albums.toArray()
        };
    }
}
```

Reply:
And description like below.

The headers are used to send additional details along with the payload. To send custom headers, it is best to create a subtype of the relevant `http:StatusCodeResponse` record by specifying the required header. Creating a subtype helps accurately generate the OpenAPI specification which then can be used to generate the relevant clients. The type of the header can be one of `string`, `int`, `boolean`, `string[]`, `int[]` or boolean[]`.